### PR TITLE
fix: removed underscore from isLabel_ in flyout_button.ts

### PR DESCRIPTION
**Pull request by sayalikandarkar**
## The basics

- [x ] I [validated my changes](https://developers.google.com/blockly/guides/contribute/core#making_and_verifying_a_change)

### Resolves

Fixes https://github.com/google/blockly/issues/7526

### Proposed Changes

This pr renames the incorrectly labeled variable and removes underscore from it precisely

### Test Coverage

I did npm run and tested all the buttons before and after and checked the entire functionality once
I have attached a screenshot of the working functionality after making the changes

### Documentation

 I don't believe any additional documentation is required
blockly

### CODE REVIEW

File: core/flyout_button.ts [hunk 1]
```diff
@@ -227,8 +227,8 @@ export class FlyoutButton {
   }
 
   /** @returns Whether or not the button is a label. */
-  isLabel(): boolean {
-    return this.isLabel_;
+  isButtonLabel(): boolean {
```
Comment (rachel-fenichel) [inline core/flyout_button.ts:230]: Aha, this must be why we didn't make this change before. The `isLabel` method is public, so we don't want to rename it.

Instead, please change `isLabel_` -> `isFlyoutLabel`.

Sorry for the wrong instructions!

Comment (sayalikandarkar) [reply core/flyout_button.ts]: Hey Rachel,

Thank you so much for your review.
I made the necessary changes and pushed them. 
Could you please take a look once?

Thanks,
Sayali

**Comment by rachel-fenichel** [comment]
Thanks!

Two things left:
- Update the PR title to match conventional commits: "chore: rename isLabel_ to isFlyoutLabel in flyout_button.ts"
- Run `npm run format` locally to format the file, then commit the result.

**Comment by BeksOmega** [comment]
@rachel-fenichel I think this is good to go!

**Review by rachel-fenichel — APPROVED** [review]

**Comment by rachel-fenichel** [comment]
:partying_face: Thanks @sayalikandarkar!

**Comment by sayalikandarkar** [comment]
Thank you so much @rachel-fenichel for all your feedback :)
I had a great time working on blockly ❤️

**Pull request merged**